src/foolscap/slicers/decimal_slicer.py

```diff
@@ -33,7 +34,7 @@ def receiveChild(self, obj, ready_deferred=None):
         assert ready_deferred is None
         if self.value != None:
             raise BananaError("already received a string")
-        self.value = decimal.Decimal(obj)
+        self.value = decimal.Decimal(six.ensure_str(obj, 'latin1'))
 
```

Comment:
Why `latin1`? (Probably could have a comment, too?)

Reply:
Default is utf-8. In this case these are always number strings like "10000" so latin1 encoding is kind of assured. Can be left to default as well. Dont remember which context I did it specifically.